### PR TITLE
Think is in regular.

### PR DESCRIPTION
**Pull request by norbiboy1**
I worked on this project to finish it, I understand the procedures.
So i modified the main files.

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -173,3 +176,582 @@
 .headline .social-media-icons > li {
   list-style: none;
 }
+
+/* Works layout */
+.works {
+  display: flex;
```
Comment (leonardodiasb) [inline main.css]: - [ ] It is required to use display grid in the works section as it is mentioned in the Project Requirements:
- In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.

File: main.html [hunk 2]
```diff
@@ -41,5 +43,159 @@ <h1>I’m Norbert. <br>Glad to see you!</h1>
...
+                </div>
+            <p class="secondary-section">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="language-tags">
+                    <li class="html">html</li>
+                    <li class="css">css</li>
+                    <li class="javaScript">javaScript</li>
+            </ul>
+            <div class="action-button">
+                <p>See Project</p>
+            </div>
```
Comment (leonardodiasb) [inline main.html]: - [ ] 'See Project' is a button. Kindly wrap buttons in `<button>` tags.

File: main.html [hunk 3]
```diff
@@ -41,5 +43,159 @@ <h1>I’m Norbert. <br>Glad to see you!</h1>
...
+                <li><a href="https://www.linkedin.com/in/siko-norbert/"><img src="images/mobile_menu_images/Linkedin-icon.jpg" alt="Linkedin icon"></a></li>
+                <li><a href="#"><img src="images/mobile_menu_images/Vector.jpg" alt="Thumb up icon"></a></li>
+                <li><a href="https://twitter.com/NorbertSiko"><img src="images/mobile_menu_images/Twitter.jpg" alt="Twitter icon"></a></li>
+                <li><a href="https://www.microverse.org"><img src="images/mobile_menu_images/Metaverse.jpg" alt="Microverse icon"></a></li>
+            </ul>
+        </div>
+    </div>
+    <div class="normal-button">
+        <p class="get-my-resume">Get my resume</p>
+    </div>
```
Comment (leonardodiasb) [inline main.html]: - [ ] 'Get my resume' is a button. Kindly wrap buttons in `<button>` tags.

File: main.html [hunk 4]
```diff
@@ -41,5 +43,159 @@ <h1>I’m Norbert. <br>Glad to see you!</h1>
...
+                <li class="HTML"><img src="images/mobile_menu_images/HTML.jpg" class="HTML-icon" alt="HyperText Mark-up Language">HTML</li>
+                <li class="CSS"><img src="images/mobile_menu_images/CSS.jpg" class="CSS-icon" alt="Cascading Style Sheets">CSS</li>
+            </ul>
+        </div>
+    </div>
+    <div class="framework-layout">
+        <h1 class="framework-text">Frameworks</h1>
+    </div>
+    <div class="skill-list-layout">
+        <h1 class="skill-list-text">Skills</h1>
```
Comment (leonardodiasb) [inline main.html]: - [ ] This section of your portfolio is not finished yet. Kindly fix the button width, the position, and alignment of the languages images and text, the font family of the languages, and add the arrow buttons.
image
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @norbiboy1, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
...
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your project design looks amazing ✔️

## Aspects to improve ♻️

- [ ] Your Pull Request should have a descriptive title and a short summary describing introduced changes in more detail
- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [OPTIONAL] Your commit messages should be meaningful. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danielufeli — APPROVED** [review]
Hi @norbiboy1  ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

a

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️
- Good use of GitHub flow ✔️
- Clean Code ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**